### ISSUE TITLE
Can the parameters of the trained model of onlineHD be quantized into binary?

### ISSUE THREAD

**Issue by yinchen1233** [hyperdimensional-computing/torchhd#180]
After training, I get a trained onlineHD model, but the model is very large when saved. Can I reduce the model size by quantizing the model into binary?

**Comment by mikeheddes** [comment]
If you are using the `Centroid` class then you can access the weights and apply any quantization function to reduce the model size. You can use PyTorch functions to store and load the weights. When you load them you would need to convert the weights back to floats. One thing to keep in mind is that weight binarization will likely decrease the model accuracy.

**Comment by mikeheddes** [comment]
Closing due to inactivity.
